### ISSUE TITLE
Brackets Installation fails.

### ISSUE THREAD

**Issue by DevPre** [folkswithhats/fedy#567]
Brackets Installation fails on Fedora 27.

Log:
```
Gjs-Message: JS WARNING: [./app.js 535]: reference to undefined property "license"
Repository successfully enabled.
Failed to synchronize cache for repo 'mosquito-brackets', disabling.
Last metadata expiration check: 0:05:13 ago on Mon 25 Dec 2017 02:52:44 PM CET.
No match for argument: brackets
Error: Unable to find a match
Gjs-Message: JS WARNING: [./app.js 404]: Too many arguments to function GLib.timeout_add: expected 3, got 4
```

**Comment by danstewart** [comment]
@DevPre Can you still reproduce this error? I've just tried to install and it worked fine.

Worth noting we'll probably be removing the brackets plugin as the copr doesn't seem to be maintained anymore. See #575

**Comment by bddenhartog** [comment]
Sorry, I should have closed this after addressing #574.

**Comment by bddenhartog** [comment]
Duplicate of #574.
